ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -50,6 +50,13 @@ tests:
     - chain: ipi-openstack-pre
     - ref: fips-check
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      INFRA_NODE_TYPE: ci.m1.xlarge
```

Comment:
why did you add `INFRA_NODE_TYPE` ? It's only useful in the `ipi-conf-inframachineset` step registry which isn't used in our workflows (a grep shows it's used by `openshift-upgrade-aws-infra-workflow` only).

Reply:
Because the [openshift-qe-ingress-perf](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-ref.yaml) step, which uses the [Ingress-perf wrapper](https://github.com/cloud-bulldozer/e2e-benchmarking/tree/master/workloads/ingress-perf), requires at least two workers with the "infra" role/label to perform the OCP Ingress performance test.

Reply:
ok, makes sense. The problem is available resources in VEXXHOST; which is why we need to not run that too often.